src/patterns/components/inputs/angular/code/datepicker.hbs

```diff
@@ -1,20 +1,33 @@
-<sprk-input-container iconContainerClasses="sprk-b-TextInputIconContainer">
-  <label
-    class="sprk-b-Label--with-icon"
-    sprkLabel>
-    Date
+<sprk-icon-input-container iconContainerClasses="">
```

Comment:
Should iconContainerClasses="" be left off of <sprk-input-container> because it's empty or is there a reason it needs to be explicitly set to empty?